doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -237,39 +495,103 @@ Run WCS 1.0 tests
       * "0.1" for ``RESY``
 
    #. ``Options``:
-  
+
       * Ensure ``Verify that the server supports XML encoding`` is *checked*
       * Ensure ``Verify that the server supports range set axis`` is *checked*
 
    #. ``Schemas``:
 
-      * Ensure that ``original schemas`` is selected
+      * Ensure that ``The server implements the original schemas from the WCS 1.0.0 specification (OGC 03-065`` is selected
 
    #. Click ``OK``
 
-   .. image:: tewcs-1_0a.jpg
+   .. image:: ./image/tewcs-1_0a.png
 
-   .. image:: tewcs-1_0b.jpg
+   .. image:: ./image/tewcs-1_0b.png
 
-.. _commandline:
+   .. image:: ./image/tewcs-1_0c.png
 
-.. _teamengine:
-Teamengine Web Application
---------------------------
 
-The Teamengine web application is useful for analyzing results of a test run. To run the web application execute::
+Run WCS 1.1 tests
+-----------------
 
-  ant webapp
-  
-From the cite tools checkout. Once started the web app will be available at:
+#. Prepare the environment:
 
-  http://localhost:9090/teamengine
-  
-To run on a different port pass the ``-Dengine.port`` system property to ant command.
+  - Start GeoServer with the citewcs-1.1 data directory. Example:
+
+   .. code-block:: shell
+
+    cd <root of geoserver install>
+    export GEOSERVER_DATA_DIR=<root of geoserver sources>/build/cite/wcs11/citewcs-1.1
+    ./bin/startup.sh
+
+
+#. Start the test:
+
+   .. code:: shell
 
-.. include:: ./guide.rst
+     make webUI
+
+#. Go to the browser and open the teamengine `webUI <http://localhost:8888/teamengine>`_.
+
+   - click on the **Sign in** button and enter the user and password.
+
+   - after creating the session, and chosing the test, enter the following parameters:
+
+   #. ``Capabilities URL``:
+
+         http://<ip-of-the-geoserver>:8080/geoserver/wcs?service=wcs&request=getcapabilities&version=1.1.1
```

Comment:
No, won't work if there is ``service=wcs`` in the link, cause the test suite adds a ``service=WCS`` in addition to it, and makes the request fail. 
Should just be:
```suggestion
         http://<ip-of-the-geoserver>:8080/geoserver/wcs
```
The test suite is adding (duplicating) all other parameters